src/config/settings.ts

```diff
@@ -31,16 +26,17 @@ const defaultSettings = {
   },
 };
 
+export type Settings = typeof defaultSettings;
```

Comment:
Should we use some kind of hungarian notation to distinguish type names from constant and variable names?  Like `SettingsType` ?  Or is capitalizing good enough?

Reply:
Hmm, I assumed that because other things (like `Event`, `Promise`) tend to not have the `-Type` suffix, the convention was not to use them.  Just doing capitalization is probably good enough, because this isn't Basic and the variable names are case-sensitive?

Reply:
sounds good